File: lib-base/src/main/java/com/llj/lib/base/mvp/IPresenter.java
package com.llj.lib.base.mvp;

import android.arch.lifecycle.DefaultLifecycleObserver;

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/5/16
 */
public interface IPresenter extends DefaultLifecycleObserver {

    void init();

    void destroy();

}

File: lib-base/src/main/java/com/llj/lib/base/mvp/IPresenter.kt
package com.llj.lib.base.mvp

import android.arch.lifecycle.DefaultLifecycleObserver

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/5/16
 */
interface IPresenter : DefaultLifecycleObserver {
    fun destroy()
}
